README.md

```diff
@@ -1,5 +1,11 @@
 ![Emissary Dark Knight - some code just wants to watch the core burn](emissary-knight.png) 
 
+
+| Github Actions Status: |
```

Comment:
It may be better to just list out the badges instead of adding them to a table. If we keep adding more badges, the table is just going to grow vertically, not horizontally. 

```
[![Java CI with Maven](https://github.com/NationalSecurityAgency/emissary/actions/workflows/maven.yml/badge.svg?branch=master)](https://github.com/NationalSecurityAgency/emissary/actions/workflows/maven.yml) [![CodeQL](https://github.com/NationalSecurityAgency/emissary/actions/workflows/codeql-analysis.yml/badge.svg)](https://github.com/NationalSecurityAgency/emissary/actions/workflows/codeql-analysis.yml)
```

Optionally, you could add the ASL as well, not sure how crazy we want to go with badges:
```
[![Apache License](http://img.shields.io/badge/license-ASL-blue.svg)](https://www.apache.org/licenses/LICENSE-2.0) 
```